CuratedContent/CMakeTutorialAndReferenceResources.md

```diff
@@ -0,0 +1,87 @@
+# CMake Tutorial and Reference Resources
+
+<!--deck text start-->
+The CMake meta-build system is a large and powerful set of tools to create cross-platform builds from source for C, C++, Fortran and many other compiled programming languages.
+CMake continues to rapidly evolve and best practices can change from year to year (and even from release to release in some cases).
+Here we provide links and descriptions for the best tutorial and reference materials that exist for CMake to help stay on the forefront of effective CMake usage.
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe)
+#### Publication date: ???
+
+Resource information | Details 
+:--- | :--- 
+Book title | [Professional CMake](https://crascit.com/professional-cmake/)
+Authors | Craig Scott
+Publication | Self published, new editions put out after each new CMake release
+Web links | https://crascit.com/professional-cmake/
+
+Resource information | Details 
+:--- | :--- 
+Web Page | [CMake Reference Documentation](https://cmake.org/cmake/help/latest/)
+Authors | Kitware Staff
+Web links | https://cmake.org/cmake/help/latest/ (and drop-down section of all CMake versions)
+
+Resource information | Details 
+:--- | :--- 
+Book title | [Mastering CMake](https://crascit.com/professional-cmake/)
+Authors | Kitware Staff
+Publication | Year 2022, Self published, currently not versioned
+Web links | https://crascit.com/professional-cmake/
+
+The CMake meta-build tool and ecosystem (which includes the tools CTest and CPack with each CMake installation) continues to evolve and improve fairly rapidly from release to release.
+Kitware (the company that owns the primary CMake repository and provides the official releases of CMake) puts out three major releases of CMake each year (or about every four months).
+The best practices for using CMake evolve as new versions of CMake are released.
+(For example, at the time of this writing, CMake support for a feature called Generator Expressions is still rapidly evolving and can have a dramatic impact on the best way to write CMakeList.txt files.)
+Therefore, unlike other programming tools, trying to use web searches to find solutions for CMake can be ineffective; because even advice from just a year ago on Stack Overflow or Stack Exchange can be outdated.
+Therefore, this page gives advice based on 13+ years of CMake usage and hundreds of hours of searching for and reading CMake documentation, especially in the years 2021-2022.
```

Comment:
What is `this page` referring to here? Can you provide link. Is it the stuff above? Oh, wait, I think you mean this CC article.

```suggestion
The bssw.io editorial team has created this article informed by 13+ years of [advanced CMake usage](https://tribits.org/doc/TribitsDevelopersGuide.html) and hundreds of hours of searching for and reading CMake documentation, especially in the years 2021-2022.
```

Reply:
> What is `this page` referring to here?

I mean, "the article you are reading".